###### The motion controlling interface

### Action Service Goal

# The plan as a list of locations to reach.

Location[] plan

---

### Action Server Result

# The reached location when the plan has been followed.

Location final_location

---

### Action Server Feedback

# The last location reached so far.

Location reached_location
